### PR TITLE
Align gem version with the current pronto version 0.10

### PR DESCRIPTION
**Pull request by raimondasv**
@tomasv 

Dependencies upgraded

### CODE REVIEW

File: pronto-rails_migrations.gemspec [hunk 1]
```diff
@@ -20,8 +20,8 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency('pronto', '~> 0.9.0')
+  spec.add_dependency('pronto', '~> 0.10.0')
```
Comment (tomasv) [inline pronto-rails_migrations.gemspec:23]: Maybe make this one more permissive as well? `>= 0.10.0`

Comment (raimondasv) [reply pronto-rails_migrations.gemspec:23]: All pronto plugins follow the same pattern to be versioned exactly to the current pronto version :)
 
If you insist I can change it!

Comment (tomasv) [reply pronto-rails_migrations.gemspec:23]: Ok, lets keep it this way then.

**Comment by tomasv** [comment]
@raimondasv released: https://rubygems.org/gems/pronto-rails_migrations/versions/0.10.0

**Pull request merged**